docs/en/operation/User-CLI.md

```diff
@@ -1315,6 +1315,13 @@ This command will attach the database `hive_db_name` (of type `hive`) from the U
 When paths are mounted for `s3a://bucket1`, the mount option `aws.accessKeyId=abc` will be used,
 and when paths are mounted for `s3a://bucket2`, the mount option `aws.accessKeyId=123` will be used.
 
+Besides mount options, there are some additional properties with the `-o` options:
+  * `udb-hive.<UDB_PROPERTY>`: specify the UDB options for the Hive UDB. The options
+  are as follows
+    * `allow.diff.partition.location.prefix`: Whether to mount partitions that do not share
```

Comment:
Can we make this property a general udb property, and not hive specific?

Reply:
does glue allow partitions to be outside of the table path?  if it does not,  then this property probably does not make sense for glue?

Reply:
hrmmm, good question, I'm not sure how that works in glue. If it is possible to edit the glue metadata directly, then maybe?

Reply:
Both the `Property.java` files (Hive and Glue) now share the same code structure. I can move this builder code logic to their parent class, `UdbProperty.java`. We can decide later if necessary to add this property to Glue UDB.

Reply:
Tried abstracting the common code in `UdbProperty.java` but don't think that's a good idea, will leave as such. Also bumping this up, does this PR look OK for Hive UDB? I think we can decided whether to add that for Glue later.

Reply:
What challenges did you run into? I am concerned that we have two separate property classes and they will diverge further in the future.

Reply:
@yuzhu  Both the Builder pattern and the static property map (DEFAULT_KEYS_MAP variable in `Property.java`) are making it hard to abstract the code in the parent class. Don't want to make the code too complicated.

Reply:
got it @hycsam